sig-node/sig-node-contributor-ladder.md

```diff
@@ -0,0 +1,116 @@
+# SIG Node Contributor Ladder
+
+**Note:**
+This document is seeded with content based on initial review from existing SIG Node technical and sub-project leads.
+
+## Goal
+
+This document intends to define a standard for supporting community members to grow responsibility in the SIG.  The SIG has a goal to scale individual participation using transparent criteria.  It is aspirational, and should be re-evaluated to ensure we can meet the needs of the project with the resources available to participate.  It is expected that it provides the SIG a log for future evolution of decision making criteria.
+
+If you are a community member that wants to suggest changes, please do!
+
+## New Contributors
+
+SIG Node welcomes new contributors to the community, help is always desired.  Not all contributors are able to provide sustained contribution, but each contribution is always welcome.  This document intends to capture a leadership path for contributors that intend to provide sustain contribution to the SIG and its associated code base by taking on reviewer and approver responsibilities at various levels.
```

Comment:
minor spelling error change "provide sustain contribution" to "provide a sustained contribution"